packages/ui/src/common/components/Tooltip/Tooltip.tsx

```diff
@@ -213,6 +216,19 @@ export const TooltipPopupContainer = styled.div<{ isTooltipActive?: boolean; for
     z-index: -1;
   }
 
+  &.popper-light {
+    background-color: ${Colors.Black[75]};
+    border-color: ${Colors.Black[300]};
+  }
+  &.popper-light a {
+    color: ${Colors.Blue[500]};
+    font-weight: 700;
+  }
+  &.popper-light:after {
+    background-color: ${Colors.Black[75]};
+    border-color: ${Colors.Black[300]};
+  }
+
```

Comment:
To stay consistent with the codebase instead of adding this class add this in `packages/ui/src/memberships/components/ProfileComponent.tsx`:
```js
const StyledTooltip = styled(Tooltip)`
  background-color: ${Colors.Black[75]};
  border-color: ${Colors.Black[300]};
  a {
    color: ${Colors.Blue[500]};
    font-weight: 700;
  }
  :after {
    background-color: ${Colors.Black[75]};
    border-color: ${Colors.Black[300]};
  }
`
```

Then use:
```jsx
<StyledTooltip
  tooltipLinkText="Learn how to earn JOY’s"
  tooltipLinkURL="https://www.joystream.org/token/"
  placement="top-start"
>
```